### ISSUE TITLE
Failing to boot simply retro

### ISSUE THREAD

**Issue by batorni** [geaz/simplyRetro-Z5#6]
I was able to get my power circuit working successfully to power the pi once the button is pressed, and power is maintained on the Pi and screen.

However, I am unable to get any of the 'retro boy' splash screens or any other signs of a successful boot. I just get a black screen or a 'no signal' on the screen.

I've tried removing the retroPower.cfg as well as modifying the config to enable some debug, but the Pi still refuses to Initialize, and the modifications to retroPower.cfg don't add any debug

I took some pics of my various circuits. (Please excuse my poor soldering skills, this is my first large project!)

Here's the pics: https://imgur.com/a/roYWrGG

**Comment by geaz** [comment]
First question, because I can't see it on the pictures: Did you power the screen (micro USB)?

**Comment by GarGamel55** [comment]
Hello,

Is there an SD card in the Pi? I do not see it on the pics ....

**Comment by batorni** [comment]
Yep, screen is powered, I see it flash and it will go to a blue 'no signal' occasionally.  I've tried flipping the hdmi ribbon cable (because I couldn't figure out the proper orientation) and I get the blue 'no signal' pretty much right away.

**Comment by geaz** [comment]
What do you mean with

> because I couldn't figure out the proper orientation

Because the adapters should only fit in one direction.

To further test it:
```
Use another HDMI to Mini HDMI cable and test it
    -> It works: There seems to be something wrong with your "small" hdmi cable.
    -> It doesn't work: Attach the PI to another monitor and test, if you get an output.
```

**Comment by batorni** [comment]
The ribbon cable itself when it inserts into the hdmi connectors is the one I struggled with orientation on

**Comment by geaz** [comment]
Then please try the steps above to get further results to work with. :)

**Comment by batorni** [comment]
I'll pick up an HDMI -> HDMI Mini on my way home from work and give it a shot :)

I've also been thinking, and I do have a 64gb micro SD that I am using and am seeing some issues with the pi zero and trying to boot with a 64gb SD card. I'll hunt down a smaller SD card and try it out tonight as well.

**Comment by batorni** [comment]
I was able to get the pi to boot and the power cycle seems to be working properly for both on and off.

I do get an odd emulation station error, as seen here: https://imgur.com/a/2J83vTy

I ended up loading the v1.2 release as the v1.3 release was failing to power off with the button being held. I tried to pull down the repo and rebuild, but it looks like I was missing the wiringPi library and didn't look too much into getting it.

**Comment by geaz** [comment]
Glad it works now! May I ask, what fixed your problem?

> I ended up loading the v1.2 release as the v1.3 release was failing to power off with the button being held.

Thank you for the information! Will check, if I added the wrong configuration. The application (retroPower) itself should work as intended. At least on my device :)

> I do get an odd emulation station error, as seen here: https://imgur.com/a/2J83vTy
Yes, EmulationStation will exit, because of this "error". Thats why you are in the prompt after pressing 'start'. 

The message indicates, that you don't have any rom on your sd card. Am I right? Just put at least one rom file, a small GB game for example, in the proper folder (/root/roms/). EmulationStation should start correctly after that. After that, please don't forget to [expand the file system](https://github.com/geaz/simplyRetro-Z5#resize-root).

**Comment by batorni** [comment]
> May I ask, what fixed your problem?

It actually ended up being a bad HDMI cable from AliExpress. I bought a standard one locally that worked, so I ordered another ribbon cable one from Amazon.

> The message indicates, that you don't have any rom on your sd card. Am I right?

Just tried this before work, and you are right. I had put the .zip in the /root/roms/ folders, not the unzipped file. As soon as I unzipped it, emulation station started correctly and prompted me to configure my controller.

Now I just need to find a keyboard to use with the pi and I can get everything setup and configured :)
I'm almost there!
